lib/smart_note/questions.ex

```diff
@@ -80,6 +95,7 @@ defmodule SmartNote.Questions do
       tags
       |> String.split(",")
       |> Enum.map(&String.trim/1)
+      |> Enum.map(&String.downcase/1)
```

Comment:
We added this pre-ingestion data wrangling so we could have reliable information coming out of the database. Looking at it again, we would probably also want to put a uniq on this, but ultimately it's pretty low stakes. If we end up with a question that has the same tags on it it will be double-counted, which isn't great but isn't the end of the world.

e.g. right now if we did this we would end up with dupes Question -> tags -> 'tag1,TAG2,Tag1,tag2' -> 'tag1,tag2,tag1,tag2'
we also would have ended up with dupes before but we still do.